.github/workflows/cd.yml

```diff
@@ -169,7 +169,7 @@ jobs:
       run: |
         cd .github/workflows/cd-packaging-tests/bundler-parcel
         npm install @inrupt/solid-client@$VERSION_NR
-        npx parcel build index.ts
+        npx parcel@1.12.3 build index.ts
```

Comment:
```suggestion
        # Parcel version currently pinned because of
        # https://github.com/parcel-bundler/parcel/issues/5943
        npx parcel@1.12.3 build index.ts
```